test/test_utils.py

```diff
@@ -27,11 +27,14 @@ def parse_common_options(datadir=None,
   parser = argparse.ArgumentParser(add_help=False)
   parser.add_argument('--datadir', type=str, default=datadir)
   parser.add_argument('--logdir', type=str, default=logdir)
+  parser.add_argument('--lr_scheduler_type', type=str, default=None)
```

Comment:
All these options are not so general to be common.
Use the `opts` argument to pass in specific ones.

Reply:
Added these 3 items to opts in test_train_imagenet.py and removed them here